lib/admin/WP_Auth0_Admin_Features.php

```diff
@@ -11,12 +11,7 @@ class WP_Auth0_Admin_Features extends WP_Auth0_Admin_Generic {
 
 	protected $actions_middlewares = array(
 		'basic_validation',
-		'georule_validation',
```

Comment:
Deprecated SSO validation middleware and others left over from previous commit (#624)